tests/flow/new_react/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -869,11 +865,7 @@ var ReactClass = React.createClass({
   render: function(): any {
     // Any state access here seems to make state any
     this.state;
-    return (
```

Comment:
I'm not as big fan when it's the thing being returned. But I have no issue with it to keep things consistent 🙂